internal/api/apiservice/rebuild.go

```diff
@@ -184,21 +184,22 @@ func getStrategy(ctx context.Context, deps *RebuildPackageDeps, t rebuild.Target
 	return strat, rstrat, nil
 }
 
-func buildAndAttest(ctx context.Context, deps *RebuildPackageDeps, mux rebuild.RegistryMux, a verifier.Attestor, t rebuild.Target, strat rebuild.Strategy, rstrat *repoStrategy) (err error) {
+func buildAndAttest(ctx context.Context, deps *RebuildPackageDeps, mux rebuild.RegistryMux, a verifier.Attestor, t rebuild.Target, strat rebuild.Strategy, rstrat *repoStrategy, enableSyscallMonitor bool) (err error) {
 	id := uuid.New().String()
 	remoteMetadata, err := deps.RemoteMetadataStoreBuilder(ctx, id)
 	if err != nil {
 		return errors.Wrap(err, "creating rebuild store")
 	}
 	hashes := []crypto.Hash{crypto.SHA256}
 	opts := rebuild.RemoteOptions{
-		GCBClient:           deps.GCBClient,
-		Project:             deps.BuildProject,
-		BuildServiceAccount: deps.BuildServiceAccount,
-		UtilPrebuildBucket:  deps.UtilPrebuildBucket,
-		LogsBucket:          deps.BuildLogsBucket,
-		LocalMetadataStore:  deps.LocalMetadataStore,
-		RemoteMetadataStore: remoteMetadata,
+		GCBClient:            deps.GCBClient,
+		Project:              deps.BuildProject,
+		BuildServiceAccount:  deps.BuildServiceAccount,
+		UtilPrebuildBucket:   deps.UtilPrebuildBucket,
+		LogsBucket:           deps.BuildLogsBucket,
+		LocalMetadataStore:   deps.LocalMetadataStore,
+		RemoteMetadataStore:  remoteMetadata,
+		EnableSyscallMonitor: enableSyscallMonitor,
```

Comment:
nit: I've been using "Use" as the prefix. "Enable" could work but biasing towards brevity and consistency with other code, I think I'd lean towards "Use"